spark/agent/src/main/scala/za/co/absa/spline/harvester/SparkLineageInitializer.scala

```diff
@@ -91,30 +91,21 @@ object SparkLineageInitializer extends Logging {
     def createEventHandler(): Option[QueryExecutionEventHandler] = {
       val configurer = new DefaultSplineConfigurer(defaultSplineConfiguration, sparkSession)
       if (configurer.splineMode != DISABLED) {
```

Comment:
Suggestion : (configurer.splineMode != DISABLED) ? createEventHandler(configurer) : None

Reply:
I would as well like to do that, but I don't think it's a correct Scala syntax :)